src/components/Element/collapseCard.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react'
```

Comment:
It would be better to use [MUI accordion](https://mui.com/material-ui/react-accordion/) for this